File: WebClient/src/app/model/analog.ts
export class IAnalog {
    constructor(
        public id: number,
        public Name: string,
        public Description: string,
        public HardwareType: string,
        public Address: string,
        public Multiplier: number,
        public RawValue: number,
        public Units: number,
        public Value: number,
        public DeviceId: number
    ) {}
}

File: Device/DeviceController.Data/Analog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DeviceController.Data
{
    public class Analog
    {
        public int Id;
        public string Name;
        public string Description;
        public HardwareTypes HardwareType;
        public string Address;
        public double Multiplier;
        public double RawValue;
        public string Units;
        public double Value;
        public int DeviceId;
        public DateTime? CreatedAt;
        public DateTime? UpdatedAt;
    }
}
